using-mobile-extensions/adobe-mobileservices/adobe-analytics-mobile-services.md

```diff
@@ -74,13 +74,13 @@ Select **Enter Custom settings** and complete the following tasks
 
 You can find the Acquisition App ID in Mobile Services. Select your app, navigate to Manage App Settings and under SDK Acquisition Options, copy the hashed string similar to the highlighted value:
```

Comment:
Please reword as follows:
"You can find the Acquisition App ID in Mobile Services. Select your app, navigate to the Manage App Settings page, and in the **SDK Acquisition Options** section, copy the hashed string that is similar to the highlighted value:"